bitnami/wavefront-prometheus-storage-adapter/templates/_helpers.tpl

```diff
@@ -0,0 +1,94 @@
+{{/*
+Return the proper wavefront-prometheus-storage-adapter image name
+*/}}
+{{- define "wavefront-prometheus-storage-adapter.image" -}}
+{{ include "common.images.image" (dict "imageRoot" .Values.image "global" .Values.global) }}
+{{- end -}}
```

Comment:
Suggestion: use `wfpsa` instead of `wavefront-prometheus-storage-adapter` for all these macros to make it simpler. We did sth similar on Spring Cloud Dataflow where we decided to use `scdf`.

Reply:
Done